packages/icons/src/components/ChevronUp.tsx

```diff
@@ -0,0 +1,30 @@
+/** This file is auto-generated, do not edit it manually **/
+import React from 'react'
+
+import { IconWrapper, IconProps } from '../wrappers'
+
+export const ChevronUp: React.FC<IconProps> = ({ svgProps: props, ...restProps }) => {
```

Comment:
We already have ChevronDown icon at `@open-condo/icons`. You don't need to add a new one

Reply:
I've tried to use ChevronDown and flip it vertically, but custom styles was not reflecting on the icon. So, it's easier just to have an Up icon, as like Left and Right

Reply:
If you miss some icon, you can request our design team to add it. They're the source of truth